pre_commit_hooks/file_contents_sorter.py

```diff
@@ -18,19 +18,18 @@
 
 
 def sort_file_contents(f):
-    before = tuple(f)
+    before = [line.strip(b'\n\r') for line in f if line.strip()]
```

Comment:
Oh wait actually I don't think this is an issue after all. We compare before vs after to determine whether file needs to be updated. However, the `before` list has blank lines removed by this list comprehension, so we're actually comparing before-without-blank-lines to after-without-blank-lines.

So if the input file has the lines in order but with extra lines scattered throughout, this code will see that file as passing and not edit it. 

So I don't think any changes are required here... did I miss something or does ^ make sense?